src/gc/unix/gcenv.unix.cpp

```diff
@@ -530,6 +532,9 @@ uint32_t GCToOSInterface::GetCurrentProcessCpuCount()
     if (count == 0 || count > 64)
         count = 64;
 
+    if (cgroup.GetCpuLimit(&cgroupCpuCount)){
```

Comment:
(I was going to push a fix for the broken build but I see you just dit it here https://github.com/dotnet/coreclr/commit/8b8231291ded137c49632ef808bbbdc9a500507b)